pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py

```diff
@@ -779,8 +778,11 @@ def create_pulp3_content(self):
 
         """
         treeinfo_serialized = self.get_treeinfo_serialized()
```

Comment:
Should this have even been called if the content can't be created?  At minimum the docstring for ```get_treeinfo_serialized()``` needs to make it clear that it can fail.

Reply:
I'll add to the function-doc. DistributionTree-creation only happens if you have a treeinfo - you can't not-create it, until after you know whether get_treeinfo_serialized() worked (or not). This is just one more never-happen/edge-case where we flag a warning and continue, rather than abort the migration completely in the face of bad data (which is what happens w/out this fix, even if skip-corrupted=true)